api/routes/progress_routes.py

```diff
@@ -2,7 +2,7 @@
 
 from db.queries.scores.queries import get_scores_for_app_sub_crit
 from config.mappings.mapping_parts.scored_criteria import scored_criteria
-
+from flask import current_app
```

Comment:
i think we can remove this as not being used?

Reply:
Yea will do